packages/react/src/primitives/types/phoneNumberField.ts

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 
 import { SelectFieldProps } from './selectField';
-import { TextInputFieldProps } from './textField';
+import { TextFieldProps } from './textField';
 
 interface optionalPhoneNumberFieldProps
-  extends TextInputFieldProps,
+  extends TextFieldProps,
     CountryCodeFieldProps {
```

Comment:
Unrelated to this PR, but should this be `DialCodeFieldProps`? @jacoblogan

Reply:
Will add to type cleanup task in asana